### PR TITLE
TAJO-2060: Upgrade geoip-api-java library

### PR DESCRIPTION
**Pull request by combineads**
Tajo use the dspace-geoip library, version is 1.2.3.
Now geoip data doesn't work at 1.2.3 version.
We need change 1.2.14 from 1.2.3 version.

### CODE REVIEW

File: tajo-core-tests/src/test/java/org/apache/tajo/util/TestGeoIPUtil.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    File tmpFile = File.createTempFile("GeoIP", ".dat");
+
+    try (InputStream is = new URL("http://www.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz")
+        .openConnection().getInputStream();
+        OutputStream out = new FileOutputStream(tmpFile)) {
+      IOUtils.copy(new GZIPInputStream(is), out);
+    }
+    return tmpFile;
+  }
+}
```
Comment (jinossy) [inline tajo-core-tests/src/test/java/org/apache/tajo/util/TestGeoIPUtil.java]: In my opinion, we don’t need to tests of third-party module.

File: tajo-core/pom.xml [hunk 2]
```diff
@@ -262,9 +262,9 @@
       <artifactId>commons-codec</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.dspace.dependencies</groupId>
-      <artifactId>dspace-geoip</artifactId>
-      <version>1.2.3</version>
+      <groupId>com.maxmind.geoip</groupId>
+      <artifactId>geoip-api</artifactId>
+      <version>1.2.14</version>
```
Comment (jinossy) [inline tajo-core/pom.xml]: Could you change to latest version? and this dependency scope should be changed to “provided”.

**Comment by jinossy** [comment]
@combineads 
Thanks for your contribution.
I left some comments

**Comment by combineads** [comment]
I’ve update the patch that reflects your suggestions. @jinossy 
Thank you for comment.

**Comment by jinossy** [comment]
+1 LGTM!
Test failure is not related this PR

**Pull request closed without merging**